### PR TITLE
DEV-123 Remove all previous Roadmap code from dev branch

### PR DESCRIPTION
**Pull request by QifanSun1128**
**Description**
Currently roadmap page is reachable from the main page, which is largely incomplete. The roadmap code is also embedded in the dev branch.

**Implementation**
Delete the folder "roadmap", and related parts on "mode" in files from dashboard folder.

**Testing**
Locally tested on my computer. Roadmap page is not reachable from main page any more.

### CODE REVIEW

File: lib/components/dashboard/index.tsx [hunk 1]
```diff
@@ -92,56 +90,6 @@ const Dashboard: React.FC<Props> = ({ mode }) => {
   // State Setup
   const [showNotif, setShowNotif] = useState<boolean>(true);
   const [formPopup, setFormPopup] = useState<boolean>(false);
-  // const [showMobileMenu, setShowMobMenu] = useState(false);
```
Comment (josephined21) [inline lib/components/dashboard/index.tsx:95]: hi fyona~ i don't think this commented out code (lines 95 - 144) is related to roadmap, do you mind re-adding it for now?

File: lib/components/loginTest.tsx
Comment (josephined21) [inline lib/components/loginTest.tsx:1]: i don't think this file is related to roadmap, you can leave it!

File: lib/resources/GenerateNewPlan.tsx
Comment (josephined21) [inline lib/resources/GenerateNewPlan.tsx:1]: you can keep these comments as well ^.^

File: lib/resources/majors.tsx
Comment (josephined21) [inline lib/resources/majors.tsx:1]: you can keep these comments too!

**Review by josephined21 — CHANGES_REQUESTED** [review]
hi fyona! requested some changes, mostly just comments/files you don't need to delete :) tysm~

File: lib/components/dashboard/index.tsx (hunk 1 above)
Comment (QifanSun1128) [reply lib/components/dashboard/index.tsx:95]: Screen Shot 2023-04-16 at 4 30 47 PM
Hi Josephine! When I add it back it says the values are unused so I'm not sure whether I need to add it back

File: lib/resources/majors.tsx
Comment (QifanSun1128) [reply lib/resources/majors.tsx:1]: > hi fyona! requested some changes, mostly just comments/files you don't need to delete :) tysm~

Hi Joseohine! Just made some new changes based on your request, could you please take a look? tyty

**Review by aryamanshodhan — DISMISSED** [review]
Please fix prettier issues. Have you tested this to make sure none of the features on the website break?

**Review by josephined21 — APPROVED** [review]
looks great! ty fyona :)

**Review by mpark63 — APPROVED** [review]
tested locally, nothing breaks besides some unrelated console messages

**Pull request merged**